src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/ciagmigration/jpa/entity/PreviousWorkExperiencesMigrationEntity.kt

```diff
@@ -36,6 +36,9 @@ class PreviousWorkExperiencesMigrationEntity(
   @field:NotNull
   var reference: UUID? = null,
 
+  @field:NotNull
```

Comment:
This is not null in the "Migration" entity, but not in "our" entity - at least not yet. We may choose to make it mandatory after the data migration from the CIAG API.